package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@redocly/openapi",
-  "version": "1.0.0-beta.94",
+  "version": "1.0.0-beta.95",
```

Comment:
I'd suggest making a separate MR for updating the version and the documentation since we might want to release other changes in batch. Also, if you bump the version in `package.json`, don't forget to run `npm i` to reflect the changes in `package-lock.json`.

Reply:
This totally make sense for me, going to split this PR, thx)